app/Controller/CoGroupsController.php

```diff
@@ -463,7 +470,10 @@ function isAuthorized() {
     // View all existing Groups?
     $p['index'] = ($roles['cmadmin'] || $roles['coadmin'] || $roles['comember']);
     $p['search'] = $p['index'];
-    
+    if(!$this->CoGroup->Co->CoSetting->nonAdminsGroupCreate($this->cur_co['Co']['id'])) {
+      $p['search'] = ($roles['cmadmin'] || $roles['coadmin'] || $roles['couadmin']);
```

Comment:
Why is the search permission being changed? Isn't this RFE to only change the add permission?

Reply:
Haha, stop pointing me to my own comments.

I'm actually not sure why that comment went into CO-1637, CO-1606 would make more sense. I'm going to move it there.